src/mlpack/bindings/python/print_doc_functions_impl.hpp

```diff
@@ -218,6 +218,31 @@ inline std::string ParamString(const std::string& paramName, const T& value)
   return oss.str();
 }
 
+inline bool IgnoreCheck(const std::string& paramName)
+{
+  return !CLI::Parameters()[paramName].input;
+}
+
+inline bool IgnoreCheck(const std::vector<std::string>& constraints)
+{
+  for (size_t i = 0; i < constraints.size(); ++i)
```

Comment:
Pedantic style comment; I think adding ``{}`` to the outer for loop would improve the readability.